base/internal/message.hpp

```diff
@@ -24,7 +24,7 @@
 //@{
 template <typename T> inline std::string DebugPrint(T const & t);
 
-std::string DebugPrint(std::string const & t);
+std::string DebugPrint(std::string && t);
```

Comment:
Why?
```
string s;
DebugPrint(s); //< Will be called DebugPrint(T const & t) overload with streams
```

Reply:
Right, returned also the original version.